### PR TITLE
Adds support Breadcrumbs Helper

### PR DESCRIPTION
**Pull request by it-mob**
## Description
Adds support Breadcrumbs Helper

## Summarize
New Breadcrumb Helper
New BreadcrumbHelperTest

## Related Issues
#176

### CODE REVIEW

File: src/View/UIView.php [hunk 1]
```diff
@@ -14,6 +14,7 @@
  * @property \BootstrapUI\View\Helper\FormHelper $Form
  * @property \BootstrapUI\View\Helper\HtmlHelper $Html
  * @property \BootstrapUI\View\Helper\PaginatorHelper $Paginator
+ * @property \BootstrapUI\View\Helper\BreadcrumbsHelperHelper $Breadcrumbs
```
Comment (dereuromark) [inline src/View/UIView.php]: BreadcrumbsHelperHelper => one helper too much

File: src/View/Helper/BreadcrumbsHelper.php [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    protected $_defaultAttributes = [
+        'class' => 'breadcrumb'
+    ];
+
+    /**
+     * @inheritdoc
+     */
+    public function render(array $attributes = [], array $separator = [])
+    {
+        $attributes = $attributes + $this->_defaultAttributes;
```
Comment (dereuromark) [inline src/View/Helper/BreadcrumbsHelper.php]: you can also use +=

File: src/View/Helper/BreadcrumbsHelper.php [hunk 3]
```diff
@@ -0,0 +1,45 @@
+<?php
+/**
```
Comment (ADmad) [inline src/View/Helper/BreadcrumbsHelper.php]: Please get rid of this header.

File: src/View/Helper/BreadcrumbsHelper.php [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+ * Created by PhpStorm.
+ * User: ngirardet
+ * Date: 30.05.2017
+ * Time: 12:03
+ */
+
+namespace BootstrapUI\View\Helper;
+
+class BreadcrumbsHelper extends \Cake\View\Helper\BreadcrumbsHelper
```
Comment (ADmad) [inline src/View/Helper/BreadcrumbsHelper.php]: Add "use" statement at top and just use class name.

File: tests/TestCase/View/Helper/BreadcrumbsHelperTest.php [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+    public function testCrumbList()
+    {
+        $result = $this->Breadcrumbs
+            ->add('jadb')
+            ->add('admad')
+            ->add('joe')
+            ->render();
+
+        $expected = '<ol class="breadcrumb"><li><span>jadb</span></li><li><span>admad</span></li><li><span>joe</span></li></ol>';
+        $this->assertEquals($expected, $result);
```
Comment (ADmad) [inline tests/TestCase/View/Helper/BreadcrumbsHelperTest.php]: More tests needed for the various other placeholders used in templates.

File: src/View/Helper/BreadcrumbsHelper.php (hunk 4 above)
Comment (dereuromark) [inline src/View/Helper/BreadcrumbsHelper.php]: use ... as CoreBreadcrumbsHelper IMO

**Comment by josegonzalez** [comment]
I made a few of the desired changes but it still needs tests, and also it fails on php 5.4 since the BreadcrumbsHelper is new-ish.

File: src/View/UIViewTrait.php [hunk 6]
```diff
@@ -33,5 +33,8 @@ public function initializeUI(array $options = [])
         $this->loadHelper('Form', ['className' => 'BootstrapUI.Form']);
         $this->loadHelper('Flash', ['className' => 'BootstrapUI.Flash']);
         $this->loadHelper('Paginator', ['className' => 'BootstrapUI.Paginator']);
+        if (class_exists('\Cake\View\Helper\BreadcrumbsHelper')) {
```
Comment (ADmad) [inline src/View/UIViewTrait.php]: Why the class exists chexk?

Comment (josegonzalez) [inline src/View/UIViewTrait.php]: We'd have to drop php 5.4 support if we didn't include this, as the BreadcrumbsHelper was only introduced in CakePHP 3.3.6.

Comment (ADmad) [inline src/View/UIViewTrait.php]: Ah right, forgot about that.

**Pull request merged**